resources/lib/ui/http.py

```diff
@@ -31,6 +32,11 @@ def Session():
         })
         _SESSION = s
 
+        token = TokenDecoder.get_token(lambda x: __send_request(s, x, None, None, False))
```

Comment:
id suggest in respect to the users: if it works, give it to the people and refactor it afterwards without any time pressure^^